ledger/src/blockstore.rs

```diff
@@ -4423,6 +4424,8 @@ pub mod tests {
         std::{thread::Builder, time::Duration},
     };
 
+    const TEST_DEFER_REPAIR_THRESHOLD: Duration = Duration::from_millis(200);
```

Comment:
Any reason to decouple this from `DEFER_REPAIR_THRESHOLD`?

Reply:
I decoupled this to avoid dependencies between crates. The timing value for the blockstore tests should be independent of the value used for repair.